deepcell/applications/application.py

```diff
@@ -312,6 +312,54 @@ def _resize_output(self, image, original_shape):
 
         return image
 
+    def _batch_predict(self,
+                       tiles,
+                       batch_size):
```

Comment:
No need for the newlines

```suggestion
    def _batch_predict(self, tiles, batch_size):
```